README.md

```diff
@@ -739,7 +739,7 @@ end.get
 
 ## Users
 
-Many apps have a unified login that works across the mobile app and other systems. Accessing user accounts through parse-ruby-client lets you build this functionality on top of Parse.
+Many apps have an unified login that works across the mobile app and other systems. Accessing user accounts through parse-ruby-client lets you build this functionality on top of Parse.
```

Comment:
[Indeed](http://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user)